Generator/JsonGenerator.cs

```diff
@@ -127,7 +127,8 @@ internal static void Generate(MetadataReader mr, string outDir)
             {
                 string filepath = Path.Combine(outDir, api.Name + ".json");
                 using var fileStream = new FileStream(filepath, FileMode.Create, FileAccess.Write, FileShare.Read);
-                using var streamWriter = new StreamWriter(fileStream, Encoding.UTF8);
+                // Encode the stream as UTF-8 without adding the byte order mark.

```

Comment:
Visual Studio gives a warning saying:
```
Warning	SA1515	Single-line comment should be preceded by blank line	JsonWin32Generator	
```

Reply:
Oops, sorry and thanks for pointing that out! I'll update.